config/csv/config.go

```diff
@@ -33,6 +33,31 @@ func EmptyNull(emptyNull bool) ConfigFunc {
 	}
 }
 
+// MissingColumnNameAliasString set configs to rename empty column name with given string
+//  ,COL2 			// col   COL2
+// a,1.5   				a     1.5
+// b,2.25  				b    2.25
+// c,3.0`				c       3
+
+func MissingColumnNameAlias(MissingColumnNameAlias string) ConfigFunc {
+	return func(c *Config) {
+		c.MissingColumnNameAlias = MissingColumnNameAlias
+	}
+}
+
+// AddColumnIndexToDuplicateColumns configures if column names have duplicates
```

Comment:
"... if duplicate column names should have the column index appended to the column name to resolve the conflict.", or similar?